api/constants/Constants.ts

```diff
@@ -1,6 +1,7 @@
 const Constants = {
   HOSTNAME: process.env.HOSTNAME || 'localhost',
-  API_PORT: process.env.API_PORT || 3004
+  API_PORT: process.env.API_PORT || 3004,
+  TESTING: process.env.TESTING || false
```

Comment:
A little nervous about this, but we can talk about it in person later.

Reply:
10-4